scalameta/trees/shared/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -904,6 +904,7 @@ object TreeSyntax {
       case _: Mod.Implicit => kw("implicit")
       case _: Mod.Final => kw("final")
       case _: Mod.Sealed => kw("sealed")
+      case _: Mod.Open => kw("open")
```

Comment:
You're adding a printer for `Mod.Open`, but not for `Mod.Opaque` nor `Mod.Using`. Is that intentional?

Reply:
missed that, good catch :+1: